### DISCUSSION TITLE
Xfreerdp unable to disable wallpaper

### DISCUSSION THREAD

**Discussion by cmuyaol** [FreeRDP/FreeRDP#8536] [Q&A] [answered]
Hi,
I am trying to disable the wallpaper in Xfreerdp with the following command:
`xfreerdp /u:xxx /p:xxx /v:x.x.x.x -wallpaper`
But I found wallpaper is disabled when I specify the bpp to an arbitrary value like this:
`xfreerdp /u:xxx /p:xxx /v:x.x.x.x -wallpaper /bpp:8`
Can anyone help explain this issue?

**Comment by akallabeth** [comment, marked as answer]
the option is only a flag that is sent to the server.
if the server respects that setting or not depends on the RDP server.
